src/main/java/com/cactus/adapters/ClassAuthAdaptor.java

```diff
@@ -0,0 +1,45 @@
+package com.cactus.adapters;
+
+import com.cactus.data.EntityRepository;
+import com.cactus.entities.User;
+import java.util.Collection;
+import java.util.HashMap;
+
+/***
+ * Represents ClassAuthAdapter which implements AuthAdapter interface.
+ */
+
+public class ClassAuthAdaptor implements AuthAdapter{
+
+    public EntityRepository repository;
+
+
+    @Override
+    public Response login(String username, String password) {
+        Collection<User> users = this.repository.getAllUsers();
+        for (User user : users) {
+            if (user.getUsername().equals(username) & user.getPassword().equals(password)) {
+                HashMap<String, String> credential = new HashMap<String, String>();
+                credential.put(username, password);
+                return new Response(Response.Status.OK, credential);
```

Comment:
In accordance with the AuthAdapter method documentation, on successful login the Response payload should be the following dictionary
```
{
    "userid" : PUT_ID_HERE,
    "name" : PUT_NAME_HERE
}
```